centreon-open-tickets/www/modules/centreon-open-tickets/providers/Ivanti/IvantiProvider.class.php

```
@@ -0,0 +1,835 @@
+            'query_endpoint' => $endpoint,
+            'method' => 0, // GET par défaut, mais sera remplacé par PATCH
+            'custom_request' => 'PATCH',
+            'headers' => [
+                'Authorization: rest_api_key=' . $this->getFormValue('apiKey'),
+                'Content-Type: application/json',
+            ],
+            'postFields' => json_encode($data),
+        ];
+
+        try {
+            $response = $this->curlQuery($info);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)